### ISSUE TITLE
Find all CompleteMoab objects in prescat with checksum validation errors

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_catalog#1362]
We want to know:

1. how many there are
2. their druids.
for each druid:
- storage root
- complete moab status
- status details

A csv file would be great;  a rake task would be great;  when writing code, think about the same code possibly being executed as a ReST API call (with output being sent as body of message)

We're hoping these lists aren't super long.

@jmartin-sul may have a db query handy from working on #1324.

**Comment by jmartin-sul** [comment]
@jermnelson, i'm using the queries [here](https://github.com/sul-dlss/preservation_catalog/tree/master/db#how-many-moabs-on-each-storage-root-are-status--ok) as a start for #1324.  thought you might also find that query (and that doc in general) helpful.

**Comment by jmartin-sul** [comment]
also useful:  https://github.com/sul-dlss/preservation_catalog/tree/master/db#which-objects-arent-in-a-good-state

```ruby
CompleteMoab.joins(:preserved_object, :moab_storage_root).where.not(status: :ok).order('complete_moabs.status asc, moab_storage_roots.storage_location asc').pluck(:status, :storage_location, :druid)
```

here's some example CSV writing ruby code:
https://github.com/sul-dlss/preservation_catalog/blob/master/app/services/reporter.rb

you could just put the result of that query in a CSV via rails console, if you'd like.  in terms of a more formal rake task, i _believe_ that's covered by #1318, which @aaron-collier is working on at present.  so i don't know that this issue requires a PR/new code in pres cat (which is to say, i think this issue overlaps/duplicates #1318 a bit).  if you were to write code for this, i'd suggest waiting for #1318 to get merged, and then adding an "all roots" version of that or something, so as to keep from conflicting with that WIP.

**Comment by jermnelson** [comment]
@jmartin-sul @ndushay - if we documented in the README on how to do this query in the Rails console , would that be sufficient?

**Comment by jmartin-sul** [comment]
> if we documented in the README on how to do this query in the Rails console, would that be sufficient?

i don't think that's necessary, since it's already one of the queries documented in the DB README (and since the main README is already pretty long).

~~i think the only further action i'd take under this ticket (if any) is attaching the results from that query here as a CSV.  otherwise, i think the same ground is already covered by #1324, #1325, and #1297.~~ actually, i see #1297 already has a list, though it's a bit stale (it has 513 according to the issue description, but there were 519 when i looked yesterday).  i'm inclined to close this ticket without further action.  any attached list will eventually grow stale, and it's easy enough to run the query to find objects to fix when we get started on #1297.  i think this ticket is duplicative (unless i'm missing part of the intent, in which case @ndushay is welcome to reopen, though clarification would be helpful).

re: "when writing code, think about the same code possibly being executed as a ReST API call", i think that's better handled as part of #1320, which we aren't prioritizing highly at the moment.
